habitat-lab/habitat/datasets/rearrange/modify_episodes_for_object_rearrange.py

```diff
@@ -664,9 +518,14 @@ def add_cat_fields_to_episodes(
     parser.add_argument(
         "--target_episodes_tag", type=str, default="categorical_rearrange_easy"
     )
+    parser.add_argument(
```

Comment:
Add some documentation about what this is for.

Reply:
Added